src/synology_dsm/api/photos/__init__.py

```diff
@@ -127,7 +133,9 @@ async def download_item(self, item: SynoPhotosItem) -> bytes | None:
     async def download_item_thumbnail(self, item: SynoPhotosItem) -> bytes | None:
         """Download the given items thumbnail."""
         raw_data = await self._dsm.get(
-            self.THUMBNAIL_API_KEY,
+            self.THUMBNAIL_FOTOTEAM_API_KEY
+            if item.owner_user_id == 0
+            else self.THUMBNAIL_API_KEY,
```

Comment:
we should avoid such multiline attribute associations, just set a helper variable before, like:
```python
download_api = self.THUMBNAIL_API_KEY
if item.is_shared:
    download_api = self.THUMBNAIL_FOTOTEAM_API_KEY
```